hash_practice/exercises.py

```diff
@@ -1,19 +1,48 @@
 
+from array import array
+from distutils.log import error
+import string

```

Comment:
It looks like these `import` statements didn't get used (they were probably added by an overzealous IDE), so you can remove them